lib/Dialect/Verif/Transforms/PrepareForFormal.cpp

```diff
@@ -0,0 +1,69 @@
+//===- FlattenWires.cpp - Flattens all wires ------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//===----------------------------------------------------------------------===//
+//
+// Replace all wires with their inputs.
+//
+//===----------------------------------------------------------------------===//
+
+#include "PassDetails.h"
+#include "circt/Dialect/HW/HWOps.h"
+#include "circt/Dialect/HW/HWTypes.h"
+#include "circt/Dialect/Verif/VerifDialect.h"
+#include "circt/Dialect/Verif/VerifOps.h"
+#include "circt/Dialect/Verif/VerifPasses.h"
+#include "circt/Support/Namespace.h"
+#include "circt/Support/Naming.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#include "mlir/Pass/Pass.h"
+
+using namespace circt;
+using namespace mlir;
+using namespace verif;
+
+namespace {
+/// Flatten wires by unconditionally replacing them with their inputs
+/// Wires create an alias for a set of operations, they are usually removed
+/// through canonicalization at some point.Some wires are however
```

Comment:
```suggestion
/// through canonicalization at some point. Some wires are however
```
(idc how many spaces but should have some after a sentence-terminating '.')

"Flatten" seems a little overloaded -- not here, but across other passes (FlattenIO, FlattenModules, ..).  Works for me but just a small note.